README.md

```diff
@@ -14,10 +14,18 @@ a) On Amazon Linux 201509 AMI
 
 b) On Ubuntu 14.04 AMI
 
-> sudo apt-get update
+> sudo apt-get update 
 
 > sudo apt-get install libevent-dev gcc g++ make autoconf libsasl2-dev
 
+c) Check Automake Version
+
+> sudo automake --version 
+
+> If you are not running version 1.14.x you will need to update `<root>/configure` file's variable `am__api_version='1.14'` to match your version of automake.
+
+> sudo autoreconf -vfi (This will [remake build system files](http://www.tutorialspoint.com/unix_commands/autoreconf.htm) in the directory trees driven by configure.ac) 
```

Comment:
Fixed, no need to check your version of automake. Removed the configure and Makefine.in and when autoreconf is run it will regenerate the files based upon the version of automake installed on the machine.